spec/controllers/rents_controller_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+describe RentsController do
+  let!(:rents) { create_list(:rent, 4) }
+
+  describe 'GET #index' do
+    context 'With an unauthenticated user' do
+      it 'returns an unauthorized status' do
+        get :index
+        expect(response).to have_http_status(:unauthorized)
+      end
+    end
+
+    context 'With an authenticated user' do
+      include_context 'Authenticated User'
+
+      it 'without params returns all rents' do
```

Comment:
All examples should start with a verb. If you need to use `with`, `when` or `without` as your first word, maybe you are missing a context.